gammapy/scripts/spectrum_pipe.py

```diff
@@ -1,23 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (print_function)
-from ..utils.scripts import get_parser, set_up_logging_from_args
-from ..obs import DataStore, ObservationTable
-from ..irf import EnergyDispersion, EnergyDispersion2D
-from ..irf import EffectiveAreaTable, EffectiveAreaTable2D
-from ..data import CountsSpectrum, EventList
-from ..spectrum import EnergyBounds, Energy
-from ..background import ring_area_factor
-from astropy.coordinates import Angle, SkyCoord
+from gammapy.spectrum.spectrum_analysis import SpectrumAnalysis
+from ..utils.scripts import get_parser, set_up_logging_from_args, write_yaml, read_yaml
 import logging
 import numpy as np
+from IPython import embed
```

Comment:
This line currently makes travis-ci fail:
https://travis-ci.org/gammapy/gammapy/jobs/84525573#L1054

Could you please remove it and push to this branch, so that we see if it's "mergeable" or if there are other issues coming up on travis-ci?